vc/vc_test.go

```diff
@@ -477,15 +477,22 @@ func TestVerifiableCredential_ValidAt(t *testing.T) {
 	hhh := time.Date(2001, 0, 0, 0, 0, 0, 0, time.UTC)
 
 	// no validity period is always true; includes missing IssuanceDate(.IsZero() == true)
-	assert.True(t, VerifiableCredential{}.ValidAt(time.Now()))
+	assert.True(t, VerifiableCredential{}.ValidAt(time.Now(), 0))
```

Comment:
test with non-zero skew?

Reply:
(now the argument could just be ignored, who knows)

Reply:
added

Reply:
It will be ignored, there are no nbf/exp values set. This only validates it is within the valid window for the given values, not if a value is set.